base/context.go

```diff
@@ -46,6 +47,9 @@ type Context struct {
 	// Required unless Insecure is true.
 	Certs string
 
+	// KeySize is the key size in bits for certificate keys.
+	KeySize int
```

Comment:
We've put all flags in a context, not necessarily `base.Context`. And even the all flags in context bit isn't great. There is very little reason for this to be here. Let's place it closer to its usage.